ush/global_ens/evs_global_ens_atmos_grid2grid.sh

```diff
@@ -489,7 +503,22 @@ if [ $verify = precip ] ; then
             echo "export err=\$?; err_chk" >> run_${modnam}_ccpa${apcp}_valid_at_t${vhour}z_${metplus_job}.sh
           fi
         fi
-        if [ $metplus_job = EnsembleStat ]; then
+
+        # Indicate sub-task is completed for restart
+	echo ">$COMOUTsmall_precip/run_${modnam}_ccpa${apcp}_valid_at_t${vhour}z_fhr${lead_hr}_${metplus_job}.completed" >> run_${modnam}_ccpa${apcp}_valid_at_t${vhour}z_${metplus_job}.sh
```

Comment:
It looks like this is writing directly to COMOUTsmall. We can only copy files there.

Can the restart file get written to DATA, and then `if [ $SENDCOM = YES ] ; then` copy to COMOUTsmall?

Reply:
Added.